Example.Android/MainActivity.cs

```diff
@@ -36,9 +50,15 @@ protected override void OnCreate(Bundle savedInstanceState)
             var app = new App();
             MobileCoreAndroid.Init(app.GetType().Assembly,ApplicationContext);
             SecurityService.InitializeService();
-            IAuthService authService = AuthService.InitializeService();
-            var authConfig = AuthenticationConfig.Builder.RedirectUri("org.aerogear.mobile.example:/callback").Build();
-            authService.Configure(authConfig);
+
+            if (SilentlyRun(() => AuthService.InitializeService()))
```

Comment:
not sure I like this idea. Should we consider using the core module to get the configuration, or just use a try/catch here?

Reply:
When you say the configuration are you talking about the mobile app config (mostly redirect ui) or are you talking about the service conifg (the keycloak urls etc)?

Reply:
@wei-lee @secondsun The issue here is that the InitializeService will throw an exception if the serviceconfiguration is not present.

If we manually check for the configuration to be present, then we must put a `custom` if for any service we want to check. Moreover, we need to put the knowledge of what configuration to be checked inside the app (currently that knowledge is mainly inside the service itself).

If we put a `try/catch` then we will need to put every service we add inside a try catch making the code cluttered: 
```csharp
try
{
   AuthService.InitializeService();
} 
catch 
{ // do nothing
}
try
{
   OtherService.InitializeService();
}
catch
{  //do nothing
}
```

I believe the `SilentlyRun` approach is the cleanest solution (we can log the exception).
Maybe we can rename it to `FailSafeRun`.

Reply:
Oh wait, you mean for if the service config is missing.  Derp

Reply:
@ziccardi ok, right, I think I understand now. Thanks for the explanation.